### ISSUE TITLE
Issue with MTA auto updater when joining nightly server

### ISSUE THREAD

**Issue by Einheit-101** [multitheftauto/mtasa-blue#351]
Description
MTA auto updater does re-download old MTA versions when attempting to join a server that
is having a nightly version set as minimum. 

1. Install stable MTA
2. Set default build type for Auto updater in settings/advanced
3. Try joining SAAW WW2, MTA will ask you to download an update 
4. Press ok to install update, restart MTA
5. MTA will ask you again to install an update when trying to join my server, forever
6. You have to set "nightly" as update build type in order to end this infinite loop

Expected behavior:
MTA should not repetitive download the same useless update when trying to join a server
that is nightly while your set build type is default. 
Either download the nightly anyway or implement a prompt that asks:
"Would you like to change your update build type to nightly?"
Players get confused why MTA asks/downloads updates all the time. 

Running the latest MTA 1.5.5 nightly r14168, server and Client

**Comment by ccw808** [comment]
Your server has a min client requirement of 1.5.5-9.13987. Do any of your resources have <min_mta_version client="1.5.5-9.13987"> in the meta.xml?

**Comment by Einheit-101** [comment]
Yes, i usually apply the same minclientversion to mtaserver.conf and at least 1 resource.

**Comment by ccw808** [comment]
The problem is the updater can only get 1.5.5-9.13968.0 (as shown at https://nightly.mtasa.com/ver/)
Why have you set 1.5.5-9.13987 ?

**Comment by Einheit-101** [comment]
Because of dxDrawCircle, and i even want to Upgrade to 14190 because of setElementAngularVelocity.

I think MTA should inform players that their Update build type is below the minimum Client Version of the target Server, instead of re-download the same useless Update again and again.

My server always is one of the first using new Features, the same happened with custom animations shortly after their introduction.
